ArchiSteamFarm/Json/Steam.cs

```diff
@@ -702,6 +702,12 @@ internal sealed class NewDiscoveryQueueResponse {
 			private NewDiscoveryQueueResponse() { }
 		}
 
+		[SuppressMessage("ReSharper", "ClassCannotBeInstantiated")]
+		internal sealed class ProfileEditData {
+			[JsonProperty(PropertyName = "Privacy", Required = Required.Always)]
+			internal UserPrivacy Privacy;
+		}
```

Comment:
Add private `[JsonConstructor]` if you can. It's extra safety measure ensuring we won't accidentally create out-of-json instance, thanks.

Reply:
Also the property should be `readonly`.